android/src/com/mozilla/vpn/VPNLogger.kt

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package com.mozilla.vpn
+
+import android.content.Context
+import java.io.File
+import java.time.LocalDateTime
+import android.util.Log as nativeLog
+
+/*
+ * Drop in replacement for android.util.Log 
+ * Also stores a copy of all logs in tmp/mozilla_deamon_logs.txt
+*/
+class Log {
+    private var file: File
+    private constructor(context: Context) {
+        val tempDIR = context.cacheDir
+        file = File(tempDIR, "mozilla_deamon_logs.txt")
+    }
+
+    companion object {
+        var instance: Log? = null
+        fun init(ctx: Context) {
+            if (instance == null) {
```

Comment:
question: is "instance" a static thing? because if yes, we cannot Log.init(a), Log.init(b), right?

Reply:
Yes its static -so init(a) would do init create the instance and init(b) would do nothing.